qiita_db/support_files/qiita-db.dbs

```diff
@@ -48,7 +48,7 @@
 			</fk>
 		</table>
 		<table name="analysis_chain" >
-			<comment><![CDATA[Keeps track of the chain of analysis edits. Tracks what previous analysis a given analysis came from.If a given analysis is not in child_id, it is the root of the chain. ]]></comment>
+			<comment>Keeps track of the chain of analysis edits. Tracks what previous analysis a given analysis came from.If a given analysis is not in child_id, it is the root of the chain. </comment>
```

Comment:
Not sure why this lines changed, as I didn't touch this part. Anyways, I try fixing this line and for some reason dbschema is not fixing it. When I save, it removes the space again.